dts/bindings/vendor-prefixes.txt

```diff
@@ -487,6 +487,7 @@ raidsonic	RaidSonic Technology GmbH
 ralink	Mediatek/Ralink Technology Corp.
 ramtron	Ramtron International
 raspberrypi	Raspberry Pi Foundation
+rpi	Raspberry Pi Foundation
```

Comment:
There is already a `raspberrypi` entry right above this line. Please use that instead. The `raspberrypi` value is what upstream Linux is doing. See https://docs.zephyrproject.org/latest/guides/dts/bindings.html#vendor-prefixes for the rules.

Reply:
If I remember correctly, I added the extra vendor because the file names were changed from `raspberrypi,pico-*.yaml` to `rpi,pico-*.yaml`, and dtc required me to add rpi to the vendor list. I was requested to change all occurrences of `raspberrpi` to `rpi_pico`. If this should be an exception, I'll fix it.

Reply:
>  the file names were changed from raspberrypi,pico-*.yaml to rpi,pico-*.yaml

OK, looks like that was done at @carlescufi 's request here: https://github.com/zephyrproject-rtos/zephyr/pull/34835#discussion_r761928219

Unfortunately, he is wrong about this. The rules in the above link are clear:

1. file names should match compatibles
2. use upstream Linux vendor prefixes when possible

So this PR should be using the `raspberrypi` vendor prefix and the files should be `raspberrypi,foo.yaml` for compatible `raspberrypi,foo`. Sorry for the confusion.